tests/test.c

```diff
@@ -6486,6 +6497,99 @@ rd_kafka_resp_err_t test_AlterConfigs_simple(rd_kafka_t *rk,
         return err;
 }
 
+/**
+ * @brief Delta Incremental Alter configuration for the given resource,
+ *        overwriting/setting the configs provided in \p configs.
+ *        Existing configuration remains intact.
+ *
+ * @param configs 'const char *name, const char *value' tuples
+ * @param config_cnt is the number of tuples in \p configs
+ */
+rd_kafka_resp_err_t
+test_IncrementalAlterConfigs_simple(rd_kafka_t *rk,
```

Comment:
I don't exactly why is this needed. Can you explain once?

Reply:
Added tests for the test helper, there was something to fix